src/app/app.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
```

Comment:
I guess, you forgot to create a component for the search section

Reply:
Added search component